### DISCUSSION TITLE
Remove line break option add under tool

### DISCUSSION THREAD

**Discussion by voska7** [SubtitleEdit/subtitleedit#6034] [General]
can you please consider to add "Remove line break" option under tools. in batch convert that option is available

**Comment by darnn** [comment]
Not that I'm against this, but just in case this will be of some use to you, you can select the desired subtitles in list view and click the Unbreak button by the text box, or use the keyboard shortcut for Unbreak if you've defined one (it's under List view and text box in Settings->Shortcuts).

**Comment by voska7** [comment]
i didnot know there is shortcut option available. thank you.
